index.css

```diff
@@ -0,0 +1,1026 @@
+  margin-bottom: 1rem;
+}
+
+/* Small screen breakpoint */
+@media (min-width: 600px) {
+  .flex-small {
+    flex: 1;
+    margin-bottom: 0;
+  }
+  .flex-small.half {
+    flex: 0 0 calc(1 / 2 * 100%);
+  }
+  .flex-small.one-fourth {
+    flex: 0 0 calc(1 / 4 * 100%);
+  }
+  .flex-small.three-fourths {
+    flex: 0 0 calc(3 / 4 * 100%);
+  }
+  .flex-small.one-third {
+    flex: 0 0 calc(1 / 3 * 100%);
+  }
+  .flex-small.two-thirds {
+    flex: 0 0 calc(2 / 3 * 100%);
+  }
+}
+
+/* Large screen breakpoint */
+@media (min-width: 1000px) {
+  .flex-large {
+    flex: 1;
+    margin-bottom: 0;
+  }
+  .flex-large.half {
+    flex: 0 0 calc(1 / 2 * 100%);
+  }
+  .flex-large.one-fourth {
+    flex: 0 0 calc(1 / 4 * 100%);
+  }
+  .flex-large.three-fourths {
+    flex: 0 0 calc(3 / 4 * 100%);
+  }
+  .flex-large.one-third {
+    flex: 0 0 calc(1 / 3 * 100%);
+  }
+  .flex-large.two-thirds {
+    flex: 0 0 calc(2 / 3 * 100%);
+  }
+}
+
+
+/**
+ * Helpers
+ */
+.clearfix::before,
+.clearfix::after {
+  content: ' ';
+  display: block;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+.text-left {
+  text-align: left;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.text-justify {
+  text-align: justify;
+}
+
+.block {
+  display: block;
+}
+
+.inline-block {
+  display: inline-block;
+}
+
+.inline {
+  display: inline;
+}
+
+.vertical-center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.responsive-image {
+  max-width: 100%;
+  height: auto;
+}
+
+.show {
+  display: block !important;
+}
+
+.hide {
+  display: none !important;
+}
+
+.invisible {
+  visibility: hidden;
+}
+
+.no-padding-top {
+  padding-top: 0;
+}
+
+.no-padding-bottom {
+  padding-bottom: 0;
+}
+
+.padding-top {
+  padding-top: 2rem;
+}
+
+.padding-bottom {
+  padding-bottom: 2rem;
+}
+
+.no-margin-top {
+  margin-top: 0;
+}
+
+.no-margin-bottom {
+  margin-bottom: 0;
+}
+
+.margin-top {
+  margin-top: 2rem;
+}
+
+.margin-bottom {
+  margin-bottom: 2rem;
+}
+
+.alternate-background {
+  background: #fafafa;
+  color: #404040;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.screen-reader-text {
+  clip: rect(1px, 1px, 1px, 1px);
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+}
+
+/**
+ * Buttons
+ */
+.button, a.button, button, [type=submit], [type=reset], [type=button] {
+  -webkit-appearance: none;
+  display: inline-block;
+  border: 1px solid #000000;
+  border-radius: 4px;
+  background: #405fc4;
+  color: #ffffff;
+  font-weight: 600;
+  font-family: -apple-system, BlinkMacSystemFont, Helvetica Neue, Helvetica, Arial, sans-serif;
+  font-size: 1rem;
+  text-transform: none;
+  padding: 0.75rem 1.25rem;
+  margin: 0 0 0.5rem 0;
+  vertical-align: middle;
+  text-align: center;
+  cursor: pointer;
+  text-decoration: none;
+  line-height: 1;
+}
```

Comment:
The buttons look good on the test version. Could possibly use some comments in this section to explain which buttons are which, but everything looks well designed. Nice job!

Reply:
Very true. I had to experiment on a lot of stuff to see which line does what so comments would be a great change. Thanks!